### ISSUE TITLE
All of my sensors say "Unavailable" but Arming/Disarming still works. no clue how to debug this

### ISSUE THREAD

**Issue by CitizenRacer** [XaF/qolsysgw#101]
### Before submitting this form

- [X] I have checked that there hasn't been a similar issue submitted in the past

- [X] I have updated the issue title to something relevant and clear to understand the bug I'm facing

### Description of the bug

All of the sensors on my panel now say "unavailable" in Home Assistant. They do not report anything or respond when opened/closed or motion triggered. However, communication with the panel isn't completely broken as arming/disarming the panel from HA still works.

Restarting HA and rebooting the HA box doesn't help. Some sensors come back but eventually they all go back to being unavailable.

I've attached the diagnostics file available from the Device Info page.
[mqtt-2d3d6d14cb6f9ed58fa2914cacd238ff-Qolsys Panel-e50da4ca7cde69b7f5cd9f514e3f9481.json.txt](https://github.com/XaF/qolsysgw/files/11057351/mqtt-2d3d6d14cb6f9ed58fa2914cacd238ff-Qolsys.Panel-e50da4ca7cde69b7f5cd9f514e3f9481.json.txt)

### Expected behavior

The expected behavior is for the sensors to report their actual state and not say "unavailable"

![Screenshot 2023-03-23 at 5 14 06 PM](https://user-images.githubusercontent.com/14790049/227391831-23c84845-ce2b-41de-9e5a-7d44312767b3.jpg)

### DEBUG logs

Here's a sample log entry from the debug mode. Nothing stands out to me

2023-03-23 17:11:00.688352 DEBUG qolsys_panel: call_service: mqtt/publish, {'namespace': 'mqtt', 'retain': True, 'topic': 'homeassistant/binary_sensor/garage_window/config', 'payload': '{"name": "Garage Window", "device_class": "door", "state_topic": "homeassistant/binary_sensor/garage_window/state", "payload_on": "Open", "payload_off": "Closed", "availability_mode": "all", "availability": [{"topic": "homeassistant/alarm_control_panel/qolsys_panel/availability", "payload_available": "online", "payload_not_available": "offline"}, {"topic": "homeassistant/binary_sensor/garage_window/availability", "payload_available": "online", "payload_not_available": "offline"}, {"topic": "mqtt-client/status", "payload_available": "online", "payload_not_available": "offline"}], "json_attributes_topic": "homeassistant/binary_sensor/garage_window/attributes", "enabled_by_default": true, "unique_id": "qolsys_panel_s1700a3", "device": {"name": "Qolsys Panel", "identifiers": ["qolsys_panel"], "manufacturer": "Qolsys", "model": "IQ Panel 2+", "connections": [["mac", "3c:31:78:57:90:d1"]]}}'}

**Comment by CitizenRacer** [comment]
Ok digging again, I now have duplicate sensors...

For example I have binary_sensor.back_door which is unavailable and binary_sensor.back_door_2 which is available.

**Comment by XaF** [comment]
Did you update to 1.3.0 recently?
I had done some testing to avoid that but it seems #95 might be the culprit. I made that change to make sure that sensors would keep their configuration in home assistant across changes of the panel, but I believe there might be a side effect for running installations, since the unique ID of the sensors now changed.

**Comment by CitizenRacer** [comment]
I’m sure I did. I keep everything up to date. I’ve since deleted the old ones and renamed the new ones. So I should be ok if it’s not going to repeat. Sent from my mobileOn Mar 23, 2023, at 5:28 PM, Raphaël Beamonte ***@***.***> wrote:﻿
Did you update to 1.3.0 recently?
I had done some testing to avoid that but it seems #95 might be the culprit. I made that change to make sure that sensors would keep their configuration in home assistant across changes of the panel, but I believe there might be a side effect for running installations, since the unique ID of the sensors now changed.
